centralHub/migrations/0004_twitteruser_twitter_name.py

```diff
@@ -2,22 +2,23 @@
 
 from django.db import migrations, models
 
-def populate_unique_emails(apps, schema_editor):
```

Comment:
What was this for?

Reply:
It was a script I used to fill a schema change.